src/main.js

```diff
@@ -63,7 +63,7 @@ function setup(config) {
         setTimeout(function () {
             const state = document.readyState;
 
-            if (state === 'interactive' || state === 'complete') {
+            if (config.autostart && (state === 'interactive' || state === 'complete')) {
```

Comment:
I'm seeing desired behavior for 'autostart: false'. However, this may be way userale.start() is not working.  If 'atutostart':false' and 'setup' checks for autostart === true, then if 'autostart:false', then I'm not sure how to turn userale on...